### ISSUE TITLE
`suppressDragLeaveHidesColumns = false` doesn't always work

### ISSUE THREAD

**Issue by fidian** [ag-grid/ag-grid#4043]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of ag-Grid'
[] support request => see 'Requesting Community Support'
```

https://plnkr.co/edit/X1LOGgcJ5JnSQ9Dx
__Edit:__ This link is based off an example with only the `suppressDragLeaveHidesColumns = false` changed. Original example is at https://www.ag-grid.com/javascript-grid-grouping/ under the heading Keeping Columns Visible.

**Current behavior**

The grid usually removes the grouped field, but does not sometimes.

![grouping-bug](https://user-images.githubusercontent.com/428348/91316845-ec9f9b80-e77e-11ea-92cf-2a74bb1e4df0.gif)

**Expected behavior**
The grid always removes the grouped field, but does not sometimes.

**Please tell us about your environment:**
Chrome, multiple versions, multiple OS's.

* **ag-Grid version:** 23.2.0

* **Browser:**
"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.83 Safari/537.36"
and several others

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
TypeScript

**Comment by LouisMoore-agGrid** [comment]
Hi,

Your ticket has been flagged as in_zendesk, it means that we recognize this as an issue worth investigating so we have moved it to our official support channel for customers (zendesk)

We will carry on there with the investigation and we will update you as soon as possible.

Thanks

**Comment by kiril-matev** [comment]
Hello!



Thank you for reporting this.

We have reproduced this as a bug.

We have added this to our backlog and we are tracking it with the following reference and description:
**AG-4564 Quickly drag & dropping a column into the row group panel doesn't hide it**

We try to fix bugs from one release to the next, so this should be fixed in the next release or the one after if it was raised too close to the next release date.

You can check whether this item is selected for implementation by taking a look at the NEXT RELEASE tab of the product pipeline:
https://www.ag-grid.com/ag-grid-pipeline/

I would recommend signing up for automatic release notifications for ag-Grid on github. Then, following a release, check whether the item was implemented by looking for it in the changelog page:
https://www.ag-grid.com/ag-grid-changelog/

For more information on when can you expect your ticket to be fixed please check:
https://ag-grid.zendesk.com/hc/en-us/articles/360002213531-Where-Is-My-Ticket-

Thanks again for bringing this up with us.

**Comment by fidian** [comment]
Update: version 24.0.0 is still affected by this bug.

**Comment by kiril-matev** [comment]
Hi Fidian,

This bug will be addressed as part of our next release - agGrid v24.1.0.

Please see our pipeline for information on what items are scheduled for inclusion in that release:
https://www.ag-grid.com/ag-grid-pipeline/
